### PR TITLE
Added HighScore to the Lobby

### PR DESCRIPTION
**Pull request by LMiljak**

### CODE REVIEW

File: ContextDesktop/src/main/java/com/github/migi_1/Context/main/LobbyHUDController.java [hunk 1]
```diff
@@ -32,6 +34,7 @@ public LobbyHUDController(Application app) {
         this.main = (Main) app;
         assetManager = ProjectAssetManager.getInstance().getAssetManager();
         BitmapFont titleFont = assetManager.loadFont("Interface/Fonts/myfont.fnt");
+        menuFont = assetManager.loadFont("Interface/Fonts/myfont2.fnt");
         title = new BitmapText(titleFont, false);
```
Comment (dVoorhout) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/main/LobbyHUDController.java:41]: please make the locations them final static variables

**Comment by dVoorhout** [comment]
Everything else looks good

**Comment by LMiljak** [comment]
Fixed it.

File: ContextDesktop/src/main/java/com/github/migi_1/Context/main/LobbyHUDController.java [hunk 2]
```diff
@@ -42,14 +48,26 @@ public LobbyHUDController(Application app) {
         
         addPlayers();
         addInstruction();
+        addHighScore();
+    }
+    
+    private void addHighScore() {
```
Comment (NilsHullegien) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/main/LobbyHUDController.java:57]: Missing javadoc

**Comment by LMiljak** [comment]
Fixed it

**Comment by NilsHullegien** [comment]
Looks good now, merging

**Pull request merged**